assets/src/edit-story/utils/areEventsDragging.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Determine if two mouse events are part of a dragging action or not.
+ *
+ * It can be any two mouse event such as down and up or dragstart and drag.
+ *
+ * @param {Object} firstEvent   First mouse event object.
+ * @param {Object} secondEvent  Second mouse event object.
+ * @return {boolean} Returns true iff the events are considered part of dragging
+ */
+function areEventsDragging(firstEvent, secondEvent) {
```

Comment:
Yeah, I struggled a lot with naming this one. First I called it `isDragging` but that conflicted with local variables all over. And it's not about whether one event is a drag event, but whether the two events, seen in sequence, indicates that the gesture is a drag gesture.

Maybe `isDragGesture` is a better name then, as it describes the overall action taken by the user rather than any individual event? Or even `isPartOfDragGesture`? Or actually as we're testing two events: `arePartsOfDragGesture`, but that's quite a mouthful.

Reply:
It can still be considered one event that's being determined to be dragging or not, it's just determined looking at the other event, too (dragging itself consists of different events anyway).

We discussed the naming a long time ago as well with Dima and at that point, `isMouseUpAClick` came up as the one that was the easiest to understand since the idea was that after a mouseup, it's determined if it was a click or not.

If you feel like this should be changed now, I'd vote for `isDragEvent` or `isDragGesture` -- since independent of what the input is and what we're doing inside, the goal is to figure out if what's happening is dragging or not. Thoughts?